diff --git a/src/PayfastDonationsListBuilder.php b/src/PayfastDonationsListBuilder.php
index 1234567..abcdefg 100644
--- a/src/PayfastDonationsListBuilder.php	2026-07-16 01:08:30.451170542 +0200
+++ b/src/PayfastDonationsListBuilder.php	2026-07-16 01:08:30.463293154 +0200
@@ -67,6 +67,7 @@
 
     $total = $this->getStorage()
       ->getQuery()
+      ->accessCheck(FALSE)
       ->count()
       ->execute();
 
